### PR TITLE
Use correct org.apache.pekko group id in docs

### PR DESCRIPTION
**Pull request by mdedetrich**
Accidental typo from https://github.com/apache/incubator-pekko-management/pull/69

### CODE REVIEW

File: docs/src/main/paradox/discovery/aws.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ This is a separate JAR file:
 @@dependency[sbt,Gradle,Maven] {
   symbol1=PekkoManagementVersion
   value1=$project.version$
-  group="org.apache.pekko.pekko.discovery"
+  group="org.apache.pekko.discovery"
```
Comment (pjfanning) [inline docs/src/main/paradox/discovery/aws.md:55]: we don't currently have any jars in this groupId - we use `org.apache.pekko` for every jar - see https://repository.apache.org/content/groups/snapshots/org/apache/pekko/

Comment (mdedetrich) [reply docs/src/main/paradox/discovery/aws.md]: I see, til. Fixed and pushed.

File: docs/src/main/paradox/discovery/aws.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ preview SDK. The disadvantage is that the mainstream SDK does blocking IO.
...
```
Comment (He-Pin) [inline docs/src/main/paradox/discovery/aws.md:169]: not `group="org.apache.pekko"`?

Comment (mdedetrich) [reply docs/src/main/paradox/discovery/aws.md]: Thanks, there was indeed a few more cases of this in the docs. Just fixed and force pushed.

**Comment by He-Pin** [comment]
I don't see any update in the `project/Common.scala` for `organization`  project setting

**Comment by mdedetrich** [comment]
> I don't see any update in the project/Common.scala for organization project setting, I think that's needed too.

That is being set with [sbt-apache-sonatype](https://github.com/mdedetrich/sbt-apache-sonatype). It sets the org based on  https://github.com/apache/incubator-pekko-management/blob/b5c0aca9a7275bfb51fb47e878bc886ebc37c25d/build.sbt#L4

```
[IJ]show organization
[info] discovery-consul / organization
[info] 	org.apache.pekko
[info] discovery-kubernetes-api / organization
[info] 	org.apache.pekko
[info] integration-test-aws-api-ecs / organization
[info] 	org.apache.pekko
[info] management-cluster-bootstrap / organization
[info] 	org.apache.pekko
[info] discovery-aws-api-async / organization
[info] 	org.apache.pekko
[info] integration-test-kubernetes-dns / organization
[info] 	org.apache.pekko
[info] integration-test-marathon-api-docker / organization
[info] 	org.apache.pekko
[info] management / organization
[info] 	org.apache.pekko
[info] discovery-aws-api / organization
[info] 	org.apache.pekko
[info] lease-kubernetes-int-test / organization
[info] 	org.apache.pekko
[info] integration-test-aws-api-ec2 / organization
[info] 	org.apache.pekko
[info] integration-test-local / organization
[info] 	org.apache.pekko
[info] management-cluster-http / organization
[info] 	org.apache.pekko
[info] lease-kubernetes / organization
[info] 	org.apache.pekko
[info] docs / organization
[info] 	org.apache.pekko
[info] management-loglevels-log4j2 / organization
[info] 	org.apache.pekko
[info] integration-test-kubernetes-api / organization
[info] 	org.apache.pekko
[info] discovery-marathon-api / organization
[info] 	org.apache.pekko
[info] management-loglevels-logback / organization
[info] 	org.apache.pekko
[info] integration-test-kubernetes-api-java / organization
[info] 	org.apache.pekko
[info] management-pki / organization
[info] 	org.apache.pekko
[info] organization
[info] 	org.apache.pekko
[IJ]
```

**Comment by He-Pin** [comment]
Thanks @mdedetrich

**Review by He-Pin — APPROVED** [review]
lgtm

**Pull request merged**